### PR TITLE
[EDO-166] Edit Team Popup in Team's Living Room

### PR DESCRIPTION
**Pull request by ngnmhieu**
Team can edit their infos (name, short name, logo, ..) in living room with a popup by clicking 3-dot icon in upper right corner of the avatar.

### CODE REVIEW

File: src/main/webapp/app/teams/teams-edit.component.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    cancel() {
+        this.activeModal.dismiss('Team edit cancelled');
+    }
+
+    save() {
+        this.isSaving = true;
+        this.image.name = this.team.shortName + '-logo-' + Date.now();
+        this.image.medium = this.image.large;
```
[the remark this answers is not in the document]
Comment (ngnmhieu) [reply src/main/webapp/app/teams/teams-edit.component.ts]: That's indeed the case :+1:

File: src/main/webapp/app/teams/teams-edit.component.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        this.image = {};
+        if (this.team.imageId) {
+            this.imageService
+                .find(this.team.imageId)
+                .subscribe((res: HttpResponse<IImage>) => (this.image = res.body), (res: HttpErrorResponse) => this.onError(res.message));
+        }
+    }
+
+    cancel() {
+        this.activeModal.dismiss('Team edit cancelled');
```
[the remark this answers is not in the document]
Comment (ngnmhieu) [reply src/main/webapp/app/teams/teams-edit.component.ts]: This message is not displayed anywhere, so a generic message would be enough.

**Comment by ngnmhieu** [comment]
@sberlin All change requests are resolved

File: src/main/webapp/i18n/de/team.json [hunk 3]
```diff
@@ -5,7 +5,8 @@
             "home": {
                 "title": "Teams",
                 "createLabel": "Team erstellen",
-                "createOrEditLabel": "Team erstellen oder bearbeiten"
+                "createOrEditLabel": "Team erstellen oder bearbeiten",
```
[the remark this answers is not in the document]
Comment (sberlin) [reply src/main/webapp/i18n/de/team.json]: This change now does not have any effect. Please remove all changes from the respective `team.json` files.

**Comment by ngnmhieu** [comment]
@sberlin All translations in i18n in `team.json` are rolled back.

**Pull request merged**